### PR TITLE
feat: add faction icons to the player list

### PR DESCRIPTION
**Pull request by rottencowz**
Icons are present and now added inline in the list elements.

This works locally, but I'm not sure if there's anything I need to do to make it work once published. Will the icons/ folder be bundled up? I think so - but unsure.

Icons found from: https://root.seiyria.com/.

image

### CODE REVIEW

**Comment by rottencowz** [comment]
Also vagabond currently uses the same icon for both 1 & 2. Could possibly make them different if we find another icon

File: js/app.js [hunk 1]
```diff
@@ -133,15 +133,51 @@ function randomizeGame() {
   }
 }
 
+function getIconPath(faction) {
+  var iconName = "";
+  switch (faction) {
```
Comment (haswellr) [inline js/app.js]: I think the iconNames should go into data.js: https://github.com/haswellr/AllRandomAllRoot/blob/main/js/data.js#L1. Then you can avoid the big switch and we've got all faction-specific data in one place.

**Review by haswellr — CHANGES_REQUESTED** [review]
This is like a 100x increase in prettiness

File: js/app.js (hunk 1 above)
Comment (rottencowz) [reply js/app.js]: Yea I wasn't sure if I wanted to associate the data for the game selection with the front end decision, but I think it's fine the way you suggest. I moved the iconName to properties on the data objects. Lmk

**Pull request merged**